cli/cmd/env/configure.go

```diff
@@ -105,15 +106,33 @@ func ConfigureCmd(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to update variables %w", err)
 			}
-
 			cmdutil.SuccessPrinter("Updated project variables")
+
+			if !cmd.Flags().Changed("reconcile") {
+				triggerReconcile = cmdutil.ConfirmPrompt("Do you want to reconcile deployment", "", triggerReconcile)
+			}
+
+			if triggerReconcile {
+				project, err := client.GetProject(ctx, &adminv1.GetProjectRequest{OrganizationName: cfg.Org, Name: projectName})
+				if err != nil {
+					return err
+				}
+
+				_, err = client.TriggerReconcile(ctx, &adminv1.TriggerReconcileRequest{DeploymentId: project.ProdDeployment.Id})
+				if err != nil {
+					warn.Printf("Reconcile failed. Trigger reconcile again with `rill project reconcile` if required.")
+					return err
+				}
+				cmdutil.SuccessPrinter("Triggered reconcile successfully.")
+			}
 			return nil
 		},
 	}
 
 	configureCommand.Flags().SortFlags = false
 	configureCommand.Flags().StringVar(&projectPath, "path", ".", "Project directory")
 	configureCommand.Flags().StringVar(&projectName, "project", "", "")
+	configureCommand.Flags().BoolVar(&triggerReconcile, "reconcile", true, "Reconcile deployment")
```

Comment:
Since it's going to prompt, I think putting default `false` here makes more sense